_scss/block/_site-nav.scss

```diff
@@ -63,7 +63,7 @@
 
     &--logo {
       display: inline-block;
-      max-width: 88px;
+      max-width: 100px;
```

Comment:
Make sure you've checked this on mobile; I think the logo was smaller to prevent causing the menu to wrap on small screens.